app/scripts/views/change_password.js

```diff
@@ -72,6 +72,9 @@ function (Cocktail, BaseView, FormView, AuthErrors, Template, PasswordMixin,
             return self.user.setSignedInAccount(account);
           })
           .then(function () {
+            // no response is expected, so do not wait for one.
+            self.broker.afterChangePassword(account);
```

Comment:
Something about this doesn't sit quite right with me. No response is expected from the FxDesktop broker, but that is not necessarily the case for any other brokers that implement `afterChangePassword`.